api/lib/src/lib.rs

```diff
@@ -628,6 +634,20 @@ pub fn clean_foreign_chain_address(chain: ForeignChain, address: &str) -> Result
 	})
 }
 
+#[async_trait]
+pub trait DepositMonitorApi:
+	SignedExtrinsicApi + StorageApi + Sized + Send + Sync + 'static
+{
+	async fn mark_btc_transaction_as_tainted(&self, tx_id: cf_chains::btc::Hash) -> Result<()> {
+		let _ = self
+			.submit_signed_extrinsic(state_chain_runtime::RuntimeCall::BitcoinIngressEgress(
+				pallet_cf_ingress_egress::Call::mark_transaction_as_tainted { tx_id },
+			))
+			.await;
```

Comment:
Can we return the result here instead of ignoring it?

Reply:
The return type of that `.await` is `(H256, (<Self as SignedExtrinsicApi>::UntilInBlockFuture, <Self as SignedExtrinsicApi>::UntilFinalizedFuture))`, I am now returning the first `tx_hash: H256` element.

Reply:
Yes I see, done.